features/shared/entering_parent_branch.feature

```diff
@@ -31,6 +31,7 @@ Feature: Entering a parent branch name when prompted
     When I run `git sync` and enter "2" and "1"
     Then I see "Please specify the parent branch of feature-2"
     And I see "Please specify the parent branch of feature-1"
+    And I only see the branch list once
```

Comment:
Another way to go about this could be to define in more detail how the output looks like:

```cucumber
Then I see
  """
  Feature branches can be branched directly off
  main or from other feature branches.

  The former allows to develop and ship features completely independent of each other.
  The latter allows to build on top of currently unshipped features.

  Please specify the parent branch of feature-2
  Please specify the parent branch of feature-1
  """
```

The advantage of this approach is that we avoid the step `I only see the branch list once`, which feels a bit weird.